src/test/fuzz.cpp

```diff
@@ -130,9 +136,12 @@ fuzz(std::string const& filename, el::Level logLevel,
 
 restart:
 {
+    srand(1);
```

Comment:
extract this to a function to avoid copy/paste

Reply:
Done